addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8603,7 +8603,7 @@ msgid "Recording completed"
 msgstr ""
 
 msgctxt "#19228"
-msgid "Recording deleted"
+msgid "Timer deleted"
```

Comment:
mind adding a comment where this label is used as you already did the research :) Also a contextual comment would be great, something like
#. notification after a PVR timer was deleted